### PR TITLE
[#1778] improvement(dashboard): Dashboard adds the ability to automatically format code.

### PR DESCRIPTION
**Pull request by yl09099**
### What changes were proposed in this pull request?

Introducing the front-end eslint code validation tool and formatting the front-end code.

### Why are the changes needed?

Fix: #1778 

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Existential test.

### CODE REVIEW

**Comment by xianjingfeng** [comment]
Single quotes may be more suitable for front-end developers? @yl09099

**Comment by yl09099** [comment]
> Single quotes may be more suitable for front-end developers? @yl09099

done. Format the code using the command "npx prettier . --write".

File: dashboard/src/main/webapp/.eslintignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/node_modules
+/package-lock.json
+/dist
+.DS_Store
```
Comment (xianjingfeng) [inline dashboard/src/main/webapp/.eslintignore:4]:
```suggestion
.DS_Store

```

**Comment by xianjingfeng** [comment]
> > Single quotes may be more suitable for front-end developers? @yl09099
> 
> done. Format the code using the command "npx prettier . --write".

Could you add a job for checking style in the workflow?

File: dashboard/src/main/webapp/.eslintignore (hunk 1 above)
Comment (yl09099) [reply dashboard/src/main/webapp/.eslintignore:4]: What does this mean?

**Comment by yl09099** [comment]
> > > Single quotes may be more suitable for front-end developers? @yl09099
> > 
> > 
> > done. Format the code using the command "npx prettier . --write".
> 
> Could you add a job for checking style in the workflow?



> > > Single quotes may be more suitable for front-end developers? @yl09099
> > 
> > 
> > done. Format the code using the command "npx prettier . --write".
> 
> Could you add a job for checking style in the workflow?

This format check rule is automatically triggered at compile time.

**Review by xianjingfeng — APPROVED** [review]
LGTM

**Comment by zuston** [comment]
> > Single quotes may be more suitable for front-end developers? @yl09099
> 
> done. Format the code using the command "npx prettier . --write".

Could you add this format command into the dashboard doc? @yl09099

**Pull request merged**